config/config.go

```diff
@@ -59,6 +69,31 @@ type SigningProfile struct {
 	CSRWhitelist *CSRWhitelist
 }
 
+func (oid *OID) UnmarshalJSON(data []byte) (err error) {
+	if data[0] != '"' || data[len(data) - 1] != '"' {
+		return errors.New("OID JSON string not wrapped in quotes." + string(data))
+	}
+	data = data[1:len(data) - 1]
+	parsedOid, err := parseObjectIdentifier(string(data))
+	if err != nil {
+		return err
+	}
+	*oid = OID(parsedOid)
+	log.Debugf("Parsed OID %v", *oid)
+	return
+}
+
+func (oid OID) MarshalJSON() ([]byte, error) {
```

Comment:
This should probably be replaced with 

```
fmt.Sprintf(`"%v"`, oid)
```

It's a lot shorter and more apparent what's happening, I think.